device/isis/interface.yaml

```diff
@@ -0,0 +1,94 @@
+components:
+  schemas:
+    Isis.Interface:
+      description: >-
+        Container for single ISIS interface.
+      type: object
+      required: [eth_name]
+      properties:
+        eth_name: 
+          description: >-
+            The unique name of the Ethernet interface on which ISIS
+            is running. Two ISIS interfaces cannot share the same Ethernet.         
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ethernet/properties/name'        
+        metric:
+          description: >-
+            The default metric cost for the interface.
+          type: integer
+          default: 10
+        network_type:
+          description: |-
+            The type of network link. 
+          type: string
+          enum: [broadcast, point_to_point]
+          default: broadcast
+        level_type:
+          description: |-
+            This indicates whether this router is participating in L1, L2
+            or both L1 and L2 domains on this interface.
+          type: string
+          enum: [level_1, level_2, level_1_2]
```

Comment:
shouldn't we call it `level_l1` or just `l1` ? since we have properties named `l1_settings` or `l2_settings`

Reply:
Although L1, L2 are right tech term widely but modified as
"Level-1 (L1), Level-2 (L2) or Level-1-2 (L1-2)"